web/client/epics/identify.js

```diff
@@ -162,6 +162,13 @@ export const getFeatureInfoOnFeatureInfoClick = (action$, { getState = () => { }
                         const itemId = itemIdSelector(getState());
                         const reqId = uuid.v1();
                         const param = { ...appParams, ...requestParams };
+                        const infoFormat = getState()?.layers?.flat[6]?.featureInfo?.format;
```

Comment:
This is uncorrect, here you are getting for every layer the state of the 6th layer. Moreover, you should not create a request for this layer, anyway. You can filter them on queraible layer selector (line 125, 126), and there is already a flag `queryable` for this purpose. So I think you don't have to touch this file, but simply, from the UI, set the queryable flag of the layer, when the format is selected.

Reply:
I must have pushed this by mistake. I have readjusted to: queryableLayers && queryableLayers[0] && queryableLayers[0].featureInfo && queryableLayers[0].featureInfo.format;

Reply:
With regards the `DISABLE IDENTITY` icon being different, I had a brief chat with @tdipisa and he mentioned he would discuss it... I can't recall whom exactly but I think you or Stefano. I can raise it with him again

Reply:
@offtherailz I will involve Stefano to provide a better icon for this as soon as the PR is merged.